types/index.d.ts

```diff
@@ -1622,7 +1622,7 @@ declare namespace Knex {
     nullable: boolean;
   }
 
-  interface Config {
+  interface Config<SV extends object = object> {
```

Comment:
object is not a very useful default type because you can't really do anything with it. `SV extends {} = any` would be something more convenient.

Reply:
Makes sense to me. I don't actually tend to use `object` in my own code so I'm not sure what possessed me to do it here 😄